### PR TITLE
feat: multiple balance query

### LINKED ISSUE #1429: feature: multiple balance query
Instead of

```
rly q balance harpoon
address {kujira1x9fxqdkg4rumkzrck8t3qnhm30jgfsx9ntcpas} balance {100000000ukuji} 

rly q balance theta  
address {cosmos1x9fxqdkg4rumkzrck8t3qnhm30jgfsx9zr6es6} balance {3000000uatom}
```

It would be nice if you could do

```
rly q balance(s) harpoon theta
Error: a key with name theta doesn't exist
```

### PR DESCRIPTION
**Pull request by neitdung**
close #1429 

---

Description:
- Add balances cmd to query multi chain balances
- Example query: 
```
rly q balances cosmoshub osmosis
rly q balances cosmoshub osmosis --key-name=test
```
- Example response:
image

### CODE REVIEW

File: cmd/flags.go [hunk 1]
```diff
@@ -477,6 +478,14 @@ func memoFlag(v *viper.Viper, cmd *cobra.Command) *cobra.Command {
 	return cmd
 }
 
+func keyNameFlag(v *viper.Viper, cmd *cobra.Command) *cobra.Command {
+	cmd.Flags().String(flagKeyName, "", "a key from the keychain associated with a particular chain")
+	if err := v.BindPFlag(flagMemo, cmd.Flags().Lookup(flagKeyName)); err != nil {
```
Comment (reecepbcups) [inline cmd/flags.go]:
```suggestion
	if err := v.BindPFlag(flagKeyName, cmd.Flags().Lookup(flagKeyName)); err != nil {
```

File: cmd/query.go [hunk 2]
```diff
@@ -325,6 +326,78 @@ $ %s query balance ibc-0 testkey`,
...
+				}
+
+				coins, err := relayer.QueryBalance(cmd.Context(), chain, addr, showDenoms)
+				if err != nil {
+					return err
+				}
+
+				data[addr] = coins.String()
+			}
+			// Convert the map to a JSON string
```
Comment (reecepbcups) [inline cmd/query.go]: nit: comment is redundant 
```suggestion
````

**Review by reecepbcups — CHANGES_REQUESTED** [review]

File: cmd/flags.go (hunk 1 above)
Comment (Reecepbcups) [reply cmd/flags.go:483]: i hate how tabs do not work in github suggestions ugh

Comment (neitdung) [reply cmd/flags.go]: Thank you

**Review by reecepbcups — APPROVED** [review]

**Pull request merged**